### DISCUSSION TITLE
支援2億畫素相機！聯發科5G行動平台天璣1080發表

### DISCUSSION THREAD

**Discussion by AnAnistired** [1101-GS4719-C/Python-Community-GS4719-C#757] [科技新知]
發哥新產品，性能大提升，高通會如何回應呢？但不難想像的是，蘋果：笑死，菜雞互啄
[文章連結](https://liff.line.me/1454987169-1WAXAP3K/v2/article/kEQopEq?utm_source=copyshare)

**Comment by williamyen0623** [comment]
護國神山聯發科（？

**Comment by Pietra1226** [comment]
安卓真的還不夠蘋果打，這樣反而讓蘋果牙膏爽爽擠

**Comment by evanchou1204** [comment]
感覺台灣手機比不上apple是完全不意外

**Comment by Yunhans** [comment]
會不會拍出來的影片只有1080p

**Comment by manipulateed** [comment]
差距不知道有沒有縮小

**Comment by dabbie109303501** [comment]
蘋果的晶片是真的強，其他的都超不過

**Comment by Zoe06060606** [comment]
性能的大提升 希望能縮短和其他大廠的差距
